packages/field-base/src/shadow-focus-mixin.js

```diff
@@ -31,21 +31,18 @@ export const ShadowFocusMixin = (superClass) =>
 
     /**
      * Override an event listener from `KeyboardMixin`
-     * to prevent setting `focused` on Shift Tab.
+     * to prevent focusing the host element on Shift Tab.
      * @param {KeyboardEvent} event
      * @protected
      * @override
      */
     _onKeyDown(event) {
       super._onKeyDown(event);
 
-      // When focus moves with Shift + Tab, do not mark host as focused.
-      // The flag set here will be later used in focusin event listener.
+      // When focus moves with Shift + Tab, skip focusing the host element
+      // by focusing it before the default browser focus handling runs
       if (!event.defaultPrevented && event.keyCode === 9 && event.shiftKey) {
-        this._isShiftTabbing = true;
```

Comment:
`_isShiftTabbing` was only ever used in `_shouldSetFocus`, and it is never `true` there, so it seems it's obsolete and can be removed.